src/cephadm/cephadm.py

```diff
@@ -4646,23 +4646,23 @@ def _rollback(ctx: CephadmContext) -> Any:
             raise
         except (KeyboardInterrupt, Exception) as e:
             logger.error(f'{type(e).__name__}: {e}')
-            if ctx.cleanup_on_failure:
+            if not ctx.no_cleanup_on_failure:
```

Comment:
nit: just to avoid the double negation can you switch the order of the messages by checking directly `if ctx.no_cleanup_on_failure` ?

Reply:
I've updated it to follow that format